### ISSUE TITLE
Reduce maker fee ratio and increase BSQ fees by 10%

### ISSUE THREAD

**Issue by MwithM** [bisq-network/proposals#181]
> _This is a Bisq Network proposal. Please familiarize yourself with the [submission and review process](https://docs.bisq.network/proposals.html)._

## Proposal

This proposal aims to change Bisq DAO parameters for Maker and Taker fee for BTC and BSQ as displayed in the next tables:

**Current fees**
![imagen](https://user-images.githubusercontent.com/50149324/75875947-01752500-5e15-11ea-9886-c1af80954f9a.png)

**Proposed fees**
![imagen](https://user-images.githubusercontent.com/50149324/75875974-13ef5e80-5e15-11ea-8f81-c9374ed918fd.png)

Maker fees were [already reduced](https://github.com/bisq-network/proposals/issues/65) about a year ago, but I think Bisq should increase that reduction, maybe to a point where spam inhibition is the only reason to have that fee.

### Motivation to reduce maker trading fees

Market makers at Bisq face extra costs and risks compared to takers and other exchanges. They have to: 
- Run a node to keep offers online.
- Leave their bitcoins in a hot wallet without 2FA.
- For Instant trades, they await to be "called" to settle an offer, and take care of disabling offers when AFK.
- Face extra volatility due to low volumes.
- Rely on price feeds from bitcoin average and poloniex.
- Pay a fee to remove an offer.

Some [exchanges](https://blog.deribit.com/insights/maker-taker-fees-on-crypto-exchanges-a-market-structure-analysis/) already have zero or *negative* trading fees for market makers. This is impossible for Bisq, as trading fees is also a measure against network spam, but if these markets can even work with negative fees, we shouldn't fear to reduce by half market maker fees while taker fees are increased proportionally. Bisq is addressing their users to post offers to increase liquidity, and this proposal makes it easier for them.

### Motivation to increase BSQ fees

BSQ fees are increased 10% to keep track of this [proposal](https://github.com/bisq-network/proposals/issues/173) which was rejected because it supposed a drastic increase of BSQ fees in fiat terms. I think that the general idea of keeping BSQ fees at 50% discount of BTC was [not rejected](https://github.com/bisq-network/proposals/issues/173#issuecomment-586983252) so Bisq should adjust BSQ fees to reach that goal.
In this proposal, at current prices (0.7 USD/BSQ), paying with BSQ suppose a discount of 53%, while now it's about a 60% discount. It's a steady fee increase that should not disturb traders very much.

This part of the proposal is subject to BSQ and BTC volatility. If there's general acceptance, DAO proposal with parameter changes will be submitted close to the end of proposal phase.

**Comment by m52go** [comment]
I think this is an important suggestion. Why not encourage market making with lower maker fees?

**Comment by MwithM** [comment]
Even with the volatility experimented at the last days, numbers for the proposal would stay the same.
30 day average for BSQ/BTC is 7790 (0.39 USD/BSQ). At this moment, discount is 62%, and with proposed BSQ changes discount would be a 56% using 30 day average and 47% using last price of 0.00009498 BSQ/BTC.

**Comment by cbeams** [comment]
It should be clarified here that the current and proposed maker and taker fees listed above are **per bitcoin traded**.

For example, the current maker fees are 0.002 BTC / 10 BSQ per bitcoin, meaning that when making an offer to buy or sell 0.25 bitcoin, the maker fees are (0.002×0.25) 0.0005 BTC / (10*0.25) 2.5 BSQ.

**Comment by cbeams** [comment]
I updated the title of this proposal to fix a typo and to make it more explicit.

I'm upvoting this because further incentivizing makers makes sense from the perspective of increasing liquidity. The 10% increase in overall fees makes sense to me as well, though I am generally uncomfortable with this kind of "ad-hoc" fee change. We need something more systematic and predictable, like what was proposed in https://github.com/bisq-network/proposals/issues/173. That proposal was ultimately downvoted, but we should take another shot at it based on the feedback provided, particularly with regard to putting a per-cycle / per-release cap in place, so that no one fee increase is too great a shock. There should at a minimum be a spreadsheet somewhere that codifies the formula, reflects what current rates are (in the app) and should be (in the next release). It will be important to show over time that fees go down as well as up based on this systematic approach. **Could someone raise their hand here to own putting that spreadsheet and proposal together?** I think the proposal will be much more actionable and likely to be approved if we have the spreadsheet in hand and everyone can review it and provide feedback on it ahead of voting time.

Finally, I'm upvoting this with the assumption that we will make the change known in an effective manner to our audience of current and potential makers. @m52go, if this is approved, could you ensure that happens?

**Comment by MwithM** [comment]
I'm glad DAO accepted this proposal, I was worried because of lack of discussion.

> The 10% increase in overall fees makes sense to me as well, though I am generally uncomfortable with this kind of "ad-hoc" fee change. We need something more systematic and predictable, like what was proposed in #173

The reason why a 10% increase was proposed is to make the fee increase smooth, but the idea is to consider as approved the #173 proposal. I'm not sure if the discount should be 50 or 40%, as both options were considered there, but it should be somewhere close to it, and using the 30 day average price to make it systematic and predictable.

**Comment by MwithM** [comment]
I have made this simple [spreadsheet](https://cryptpad.fr/sheet/#/2/sheet/view/9wasbqRHbCYoknAdPQdFR3cdYaYaesxTBpqJGIteX2M/) where everyone can review how BSQ discount is calculated.
My idea is that fees shoud be updated only if at the last 3 days left for the end of proposal phase the BSQ discount is over or under 45%, to make it back again at 50%, using the 30 day moving average USD/BSQ price compared to USD/BTC last price.
All the process to update trading fees should be described at wiki. I already updated the current [Trading fees](https://bisq.wiki/Trading_fees) for everyone to consult it.
